config/sync/user.role.site_manager.yml

```diff
@@ -48,6 +48,13 @@ permissions:
   - 'create stanford_news content'
   - 'create stanford_page content'
   - 'create stanford_person content'
+  - 'create su_line_1'
+  - 'create su_line_2'
+  - 'create su_line_3'
+  - 'create su_line_4'
+  - 'create su_line_5'
+  - 'create su_lockup'
+  - 'create su_lockup_options'
```

Comment:
Instead of field-based permissions perhaps deny access to the whole config form.